### PR TITLE
add 1.17 versioned docs

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -154,15 +154,20 @@ module.exports = {
...
               label: "1.17",
-              path: "1.17",
+              path: "/",
+              banner: "none",
             },
             1.16: {
               label: "1.16",
-              path: "/",
+              path: "1.16",
               banner: "none",
```
Comment (ifbyol) [inline docusaurus.config.js:171]: Should we add the `unmaintained` as we had for `1.15`?

**Review by ifbyol — COMMENTED** [review]
The build is failing, it seems that some redirection is not right

**Comment by codyjlandstrom** [comment]
> The build is failing, it seems that some redirection is not right

I'm going to submit another PR to fix that quickly.

File: docusaurus.config.js (hunk 1 above)
Comment (codyjlandstrom) [reply docusaurus.config.js:171]: Yes, good call 👍

**Comment by codyjlandstrom** [comment]
Just kidding, what I was going to do seems like a larger task. Adding a commit to fix it here.

**Review by ifbyol — APPROVED** [review]

**Pull request merged**